### ISSUE TITLE
Add localized label properties to admin1/admin2

### ISSUE THREAD

**Issue by stepps00** [whosonfirst-data/whosonfirst-data#1640]
Most admin1 (region, macroregion) and some of the admin2 (county, macrocounty) records in WOF have English label properties, but almost all records lack a localized label property.

In Brazil, for example, we can use the `wof:lang_x_official` and `wof:lang_x_spoken` properties to find that Portuguese (`por`) is the local language. We can use the localized placetype translations to string together localized labels (and backfill English when missing).

Placetype | Portuguese | English
--|--|--
**Macroregion** | Região | Region
**Region** | Estado | State
**County** | Município | Municipality

For at least the following countries, I've curated localized placetype names for all admin places in that country (and will eventually curate values for all):

Complete:
- [x] [Argentina](https://github.com/whosonfirst-data/whosonfirst-data-admin-ar/pull/1)
- [x] Austria
- [x] Belgium
- [x] [Estonia](https://github.com/whosonfirst-data/whosonfirst-data-admin-ee/9)
- [x] [Australia](https://github.com/whosonfirst-data/whosonfirst-data-admin-au/pull/4)
- [x] [Brazil](https://github.com/whosonfirst-data/whosonfirst-data-admin-br/pull/1)
- [x] [Czech Republic](https://github.com/whosonfirst-data/whosonfirst-data-admin-cz/pull/10)
- [x] [Denmark](https://github.com/whosonfirst-data/whosonfirst-data-admin-dk/pull/1)
- [x] [Egypt](https://github.com/whosonfirst-data/whosonfirst-data-admin-eg/pull/1)
- [x] [France](https://github.com/whosonfirst-data/whosonfirst-data-admin-fr/pull/1)
- [x] [Germany](https://github.com/whosonfirst-data/whosonfirst-data-admin-de/pull/4)
- [x] [India](https://github.com/whosonfirst-data/whosonfirst-data-admin-in/pull/3)
- [x] Ireland
- [x] South Korea
- [x] United Kingdom
- [x] Hungary
- [x] Japan
- [x] Philippines
- [x] [Italy](https://github.com/whosonfirst-data/whosonfirst-data-admin-it/pull/14)
- [x] [Mexico](https://github.com/whosonfirst-data/whosonfirst-data-admin-mx/pull/1)
- [x] [Norway](https://github.com/whosonfirst-data/whosonfirst-data-admin-no/pull/13)
- [x] [Poland](https://github.com/whosonfirst-data/whosonfirst-data-admin-pl/pull/11)
- [x] [Portugal](https://github.com/whosonfirst-data/whosonfirst-data-admin-pt/pull/11)
- [x] [Romania](https://github.com/whosonfirst-data/whosonfirst-data-admin-ro/pull/13)
- [x] [Russia](https://github.com/whosonfirst-data/whosonfirst-data-admin-ru/pull/1)
- [x] [Saudi Arabia](https://github.com/whosonfirst-data/whosonfirst-data-admin-sa/pull/1)
- [x] [Slovenia](https://github.com/whosonfirst-data/whosonfirst-data-admin-si/pull/11)
- [x] South Africa (previously done)
- [x] Spain (previously done)
- [x] Sweden (previously done)
- [x] Switzerland
- [x] Indonesia
- [x] [Netherlands](https://github.com/whosonfirst-data/whosonfirst-data-admin-nl/pull/22)
- [x] [Turkey](https://github.com/whosonfirst-data/whosonfirst-data-admin-tr/pull/1)
- [x] [United Arab Emirates](https://github.com/whosonfirst-data/whosonfirst-data-admin-ae/pull/1)
- [x] United States (previously done)

**Comment by stepps00** [comment]
This issue is complete for the countries listed in the initial comment.

We're planning some upcoming localization work to fill in various localization coverage for certain languages.. once that work is complete, I'll QA top-level placetypes again to ensure we haven't missed any records. We should track future work in a new issue.
